_posts/2020-11-13-quarkiverse.adoc

```diff
@@ -0,0 +1,68 @@
+---
+layout: post
+title: Quarkiverse - The Hub of Community Extensions
+date: 2020-11-13
+tags: extensions quarkiverse
+synopsis: Entering a world of extensions
+author: gastaldi
+---
+
+Quarkus is a Kubernetes Native Java stack tailored for the OpenJDK HotSpot and GraalVM, crafted from the best of breed Java libraries and standards, as well as an ever growing ecosystem of its extensions. At the beginning we were accepting all the contributed extensions in the core Quarkus repository. Eventually though it grew too large and lead to a maintenance overhead. In addition to that, in some cases it's simply not always sensible to include an extension into the core Quarkus repository.
+
+Some have since then created their own repositories external to Quarkus, but others just never happened because not everyone wants the overhead of maintaining a build setup. Even the Quarkus core developers sometimes had ideas for an extension but not solid enough to be included in the Quarkus core repository.
+
+Enter the *Quarkiverse*!
```

Comment:
the text above is talking about "Quarkiverse Hub" not "Quarkiverse"

Reply:
I think we should update the above to talk about Quarkiverse in a broader sense about enabling a ecosystem of quarkus extensions that is not just maven releases but actually gets incorporated in the tooling. 

The list the challenges we are working on solving:

* make it super easy to deploy/participate in testing ci
	- Ecosystem CI (already exists and can be used today)
    - Quarkiverse Hub (for those who don't want to maintain it themselves)
* documentation aggregation
* having a registry/npmjs/ceylon heard/etc. service/site
* updating code.quarkus.io, cli's and IDE's to utilize this registry
...more? 

and then say, in all this we are ready to introduce quarkiverse hub and the others are in progress...